desktop-widgets/CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ set(SUBSURFACE_INTERFACE
 	command_base.cpp
 	command_divelist.cpp
 	locationinformation.cpp
+	qoverload.h
```

Comment:
why add a `.h` file in the middle of the `C++` files?

Reply:
Pure nonsense, my bad.

Reply:
You have not the slightest idea how relieved I am to hear that... :-)